.github/workflows/keepalive.yml

```
@@ -0,0 +1,21 @@
+name: Keepalive
+
+on:
+  # At 1:05 AM UTC, on day 18 of the month
+  schedule:
+    - cron: '5 1 18 * *'
+  workflow_dispatch:
+
+permissions: {}
+
+jobs:
+  keepalive:
+    name: Keepalive
+    # Only run cron on the silverstripe account
+    if: (github.event_name == 'schedule' && github.repository_owner == 'silverstripe') || (github.event_name != 'schedule')
```

Does this need to be ?
```suggestion
    if: (github.event_name == 'schedule' && github.repository_owner == 'silverstripe-terraformers') || (github.event_name != 'schedule')
```

---

This is intentionally `silverstripe` to cover the case when this module is transferred to the officially supported modules. For now the condition is expected to fail.